Artillery fails unless @smithy packages are installed

From version 2.0.26 onwards (including the latest version), running Artillery fails without two `@smithy` packages being installed in addition to Artillery:
- `@smithy/config-resolver`
- `@smithy/node-config-provider`

Without both of these installed, Artillery errors. When installing the two modules, Artillery commands work.

[This commit](https://github.com/artilleryio/artillery/commit/696fcfce26c2f505bef596a055398ab149218468) seems to be cause of the issue. The example below works in version 2.0.25.

Version info:
```
Artillery: 2.0.31
```
(the same issue exists for every version from 2.0.26 onwards)

Running this command: 
```
npm install artillery@2.0.31
./node_modules/bin/artillery --help
```
(it also fails running an actual load test, but this is simpler!)

I expected to see this happen:

A description of the usage and commands.

Instead, this happened:

There are several errors printed (all the same error and variations of the same stack trace), e.g.:
```sh
(node:915397) [MODULE_NOT_FOUND] Warning: ModuleLoadError
module: @oclif/core@4.11.2
task: findCommand (quick)
plugin: artillery
root: ~/node_modules/artillery
code: MODULE_NOT_FOUND
message: [MODULE_NOT_FOUND] require failed to load ~/node_modules/artillery/lib/cmds/quick.js: Cannot find module '@smithy/node-config-provider'
Require stack:
- ~/node_modules/artillery/lib/platform/aws/aws-get-default-region.js
- ~/node_modules/artillery/lib/platform/aws-lambda/index.js
- ~/node_modules/artillery/lib/launch-platform.js
- ~/node_modules/artillery/lib/cmds/run.js
- ~/node_modules/artillery/lib/cmds/quick.js
- ~/node_modules/@oclif/core/lib/module-loader.js
- ~/node_modules/@oclif/core/lib/help/index.js
- ~/node_modules/@oclif/core/lib/errors/handle.js
- ~/node_modules/@oclif/core/lib/errors/index.js
- ~/node_modules/@oclif/core/lib/config/config.js
- ~/node_modules/@oclif/core/lib/config/index.js
- ~/node_modules/@oclif/core/lib/command.js
- ~/node_modules/@oclif/core/lib/index.js
- ~/node_modules/artillery/bin/run
```
If you install `@smithy/node-config-provider` only, it then errors for the `@smithy/config-resolver` package.

--- a/packages/artillery/lib/platform/aws/aws-get-default-region.js
+++ b/packages/artillery/lib/platform/aws/aws-get-default-region.js
@@ -1,8 +1,8 @@
-const { loadConfig } = require('@smithy/node-config-provider');
 const {
+  loadConfig,
   NODE_REGION_CONFIG_FILE_OPTIONS,
   NODE_REGION_CONFIG_OPTIONS
-} = require('@smithy/config-resolver');
+} = require('@smithy/core/config');
 const debug = require('debug')('util:aws:get-default-region');
 
 let defaultRegionAlreadyChecked = false;

--- a/packages/artillery/package.json
+++ b/packages/artillery/package.json
@@ -123,6 +123,7 @@
     "@oclif/core": "^4.8.0",
     "@oclif/plugin-help": "^6.2.36",
     "@oclif/plugin-not-found": "^3.2.73",
+    "@smithy/core": "^3.24.0",
     "@upstash/redis": "^1.36.1",
     "artillery-engine-playwright": "*",
     "artillery-plugin-apdex": "*",

